client/src/App.js

```diff
@@ -138,78 +91,38 @@ class App extends Component {
           </Navbar.Header>
         </Navbar> */}
         {withStyles}
-
-        <div className={classes.root}>
```

Comment:
These are dependent on the styles above. should be classes.root. I remember fixing this already last time the styling got messed up.